taipy/gui/gui.py

```diff
@@ -1111,10 +1111,13 @@ def __request_var_update(self, payload: t.Any):
 
     def __handle_ws_get_module_context(self, payload: t.Any):
         if isinstance(payload, dict):
+            page_path = str(payload.get("path"))
+            if page_path in {"/", ""}:
```

Comment:
```suggestion
            if page_path in ["/", ""]:
```
More efficient don't you think?

Reply:
set and list as iterable is similar i think. Set is also safer in most cases. we should move to set even in other cases where index is not important